.github/workflows/flownet.yml

```diff
@@ -79,6 +82,16 @@ jobs:
           make html
           popd
 
+      - name: Build and deploy Python package
+        if: github.event_name == 'release' && matrix.python-version == '3.6'
+        env:
+          TWINE_USERNAME: __token__
+          TWINE_PASSWORD: ${{ secrets.pypi_flownet_token }}
+        run: |
+          python -m pip install --upgrade setuptools wheel twine
+          python setup.py bdist_wheel
+          twine upload dist/*
+
```

Comment:
When the event triggering the workflow is a release, and we are using Python version 3.6 (just to ensure we are not trying to deploy multiple times), we deploy to PyPI using the standard tool `twine` (together with a `flownet` specific token secret).